docs/useMultiStateValidator.md

```diff
@@ -0,0 +1,55 @@
+# `useMultiStateValidator`
+
+Each time any of given states changes - validator function is invoked.  
+
+## Usage
+```ts 
+import * as React from 'react';
+import { useMultiStateValidator } from 'react-use';
+
+const DemoStateValidator = (s: number[]) => [s.every((num: number) => !(num % 2))] as [boolean];
+const Demo = () => {
+  const [state1, setState1] = React.useState<number>(1);
+  const [state2, setState2] = React.useState<number>(1);
+  const [state3, setState3] = React.useState<number>(1);
+  const [[isValid]] = useMultiStateValidator([state1, state2, state3], DemoStateValidator);
```

Comment:
Why is returning 2 arrays nested?

Reply:
Same as for `useStateValidator` validity state is an array where 1st element is an overall validity state, and rest is arbitrary data. It is made to make the validity state same signature over any use whether you pass extra validity data or not.

I've ommitted here the revalidate function, this line could be
```typescript
const [validity, revalidate] = useMultiStateValidator([state1, state2, state3], DemoStateValidator);
```

Reply:
Aah I didn't remember that [react-hooks-testing-library already catches errors and exposes them under `result.error`](https://react-hooks-testing-library.com/usage/advanced-hooks#errors) so it's probably related to that.